internal/cloud_provider/resource_cloud_provider.go

```diff
@@ -369,8 +390,7 @@ func buildConfig(d *schema.ResourceData) (map[string]interface{}, error) {
 	return config, nil
 }
 
-func resourceCloudProviderCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) (
-	diag.Diagnostics) {
+func resourceCloudProviderCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
Can we remove this formatting to adhere to the lint limit of 100 characters per line